js/background.js

```diff
@@ -341,7 +377,9 @@
         );
 
         Whisper.DeliveryReceipts.add({
-            timestamp: timestamp, source: pushMessage.source
+            timestamp: timestamp,
+            source: pushMessage.source,
+            confirm: ev.confirm
         });
```

Comment:
Maybe use the 'remove' event on the receipt instead of passing through the confirm method?
```
var receipt = Whipser.DeliveryReceipts.add({...});
receipt.on('remove', confirm);
```